nservicebus/messaging/publish-subscribe/index.md

```diff
@@ -31,14 +31,14 @@ Or in simpler terms
 
 ## Mechanics
 
-Depending on the features provided by a given transport there are two possible implementations of Publish-Subscribe mechanics: "Persistence based" and "Native".
+Depending on the features provided by a given transport there are two possible implementations of Publish-Subscribe mechanics: "Emulated" and "Native".
```

Comment:
> "Persistence based" => "Emulated"

I'm not sure how much people are used to those terms, but I don't see a good reason for changing it. If we insist, then I'd suggest a different name than "emulated" (like "NSB pub-sub" or "non-native").